examples/plotting/file/dynamic_color_mapping.py

```diff
@@ -44,8 +44,7 @@ def data_sloped():
 g3 = p3.scatter(x3, y3, radius=r3, fill_color=dict(field="radius", transform=mapper), fill_alpha=0.6, line_color=None)
 mapper.domain.append((g3, "radius"))
 
-color_bar = ColorBar(color_mapper=mapper, location=(0, 0), orientation="horizontal",
-    padding=0, ticker=BasicTicker(), formatter=BasicTickFormatter())
+color_bar = ColorBar(color_mapper=mapper, padding=0, ticker="auto", formatter="auto")
```

Comment:
why explicitly set `ticker` and `formatter` to their default value?

Reply:
Overlooked this case.